### ISSUE TITLE
Delayed credits for a spouse is causing total benefits to exceed 1/2 of max earners. Is this correct?

### ISSUE THREAD

**Issue by stkeros** [Gregable/social-security-tools#61]
Question to answer (male husband, female wife): can a lower earning wife whose PIA is less that half of husband, ever have a total benefit which is greater than husbands PIA/2? I think "no".

Currently in the calculator, if an older wife delays her personal benefit, the increased benefit is leading to a situation where personal benefit + eventual spouse benefit results in a benefit higher than husband's PIA/2.

For example, using the 1985 demo, with husband as self and wife as spouse, with older wife with PIA 100, and husband with default 1035 PIA. If wife delays her personal benefit to age 70, she receives PIA*1.24= $124. But when she finally gets a spousal benefit, the calculator is adding an additional $517 to her $124 for a total benefit of $541. Shouldn't the total benefit be capped at $517?

![image](https://user-images.githubusercontent.com/25359648/41809988-a3ebf456-76c4-11e8-9a2c-644d02930d78.png)

**Comment by Gregable** [comment]
I don't think so. My understanding is that the personal component of her benefit benefits from delayed credits.

So, for example, if the husband files at the wife's NRA, but the wife delays beyond the NRA, then the wife should have a higher than 1/2 total benefit. The spousal benefit itself it still <1/2 of the husband benefit, but the total is higher.

I need to see if AnyPIA can simulate some of these scenarios to verify these guesses.

**Comment by Gregable** [comment]
Sadly AnyPIA doesn't help much. It does not allow you to enter the earnings information for more than one recipient per report. So, you can specify a spouse, and it will tell you what the maximum family benefit and the 50% benefit are for that spouse, but not how that interacts with the spouse's own personal benefit.

**Comment by Gregable** [comment]
Posted my hypothetical on BH: https://www.bogleheads.org/forum/viewtopic.php?f=2&t=252385

Hopefully someone will answer and provide citations.

**Comment by stkeros** [comment]
Answered with references. Lot's of knowledgeable people on BH!

> See https://secure.ssa.gov/apps10/poms.nsf/lnx/0300615694
> 
> “RS 00615.694 DRCs in Dual-Entitlement Cases
> 
> A. Policy
> 
> If a beneficiary is entitled to his/her own RIB with DRCs and to benefits as an auxiliary/survivor, the combined payment amount is computed without consideration of the DRCs. The DRCs are then added to the RIB and that amount is subtracted from the combined payment amount to determine the amount payable as an auxiliary/survivor. See RS 00615.240 for reduced B benefits prior to A benefits.”

If you click through to the the "POMS" the terms MOE = month of entitlement and RIB = retirement insurance benefit and DRC = delayed retirement credit. It's somewhat obvious from the context.

**Comment by Gregable** [comment]
Nice. Was also just reading the responses. I suppose this means that there is a corner case to cover.

If the husband's PIA is $1,000 and the wife's PIA is $450, then if the wife files for benefits at NRA, then she will receive $500 (spousal). However, if the wife waits 3 years and files at 70, she will receive $558, since her own benefit with the 24% DRCs is higher than the spousal benefit, but the spousal benefit component will be $0.

I'll work on these changes.

**Comment by stkeros** [comment]
> If the husband's PIA is $1,000 and the wife's PIA is $450, then if the wife files for benefits at NRA, then she will receive $500 (spousal). However, if the wife waits 3 years and files at 70, she will receive $558, since her own benefit with the 24% DRCs is higher than the spousal benefit, but the spousal benefit component will be $0.

Correct. In this case her spousal benefit would be negative $58. But there are no negatives, so it goes to zero spousal benefit.

And just for reference, the POMS says "NOTE: The DRC increase can be an amount sufficient to cause the other benefit to be suspended or terminated."

I spent so much time researching this question and come up empty. Glad we have a definitive answer. I "knew" it to be true but could not "prove" it.

I also went to anypia and discovered what you did, earlier, but didn't have a chance to warn you not to try.
